3.1/alpine3.14/amd64/Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM mcr.microsoft.com/dotnet/core/aspnet:2.2-alpine3.8
+FROM mcr.microsoft.com/dotnet/aspnet:3.1-alpine3.14
```

Comment:
Do you know if 3.1-alpine will just pull the latest alpine version?

Reply:
It will pull the latest, which will be alpine3.15 in the next month or so. Because we are stating that this Dockerfile produces an Alpine 3.14 image, then it should be based on the Alpine 3.14 ASP.NET image.